actors/ppo_atari_pz.py

```diff
@@ -0,0 +1,1085 @@
+                implementation="actors.ppo_atari_pz.PPOActor",
+                config=AgentConfig(
+                    run_id=run_session.run_id,
+                    environment_specs=self._environment_specs.serialize(),
+                    model_id=model_id,
+                    model_version=version_info["version_number"],
+                ),
+            )
+
+            return cogment.TrialParameters(
+                cog_settings,
+                environment_name="env",
+                environment_implementation=self._environment_specs.implementation,
+                environment_config=EnvironmentConfig(
+                    run_id=run_session.run_id,
+                    render=False,
+                    seed=self._cfg.seed + trial_idx + iter_idx * self._cfg.epoch_num_trials,
+                ),
+                actors=[agent_actor_params],
+            )
+
+        # Run environment
+        observations = []
+        actions = []
+        rewards = []
+        dones = []
+        episode_rewards = []
+        episode_lens = []
+
+        num_updates = 0
+        total_steps = 0
+        for iter_idx in range(self._cfg.num_iter):
+            for (_, _, _, sample) in run_session.start_and_await_trials(
+                trials_id_and_params=[
+                    (f"{run_session.run_id}_{iter_idx}_{trial_idx}", create_trial_params(trial_idx, iter_idx))
+                    for trial_idx in range(self._cfg.epoch_num_trials)
+                ],
+                sample_producer_impl=self.trial_sample_sequences_producer_impl,
+                num_parallel_trials=self._cfg.num_parallel_trials,
+            ):
+                # Collect the rollout
+                (trial_observation, trial_action, trial_reward, trial_done) = sample
+
+                observations.extend(trial_observation)
+                actions.extend(trial_action)
+                rewards.extend(trial_reward)
+                dones.extend(trial_done)
+
+                episode_rewards.append(torch.vstack(trial_reward).sum())
+                episode_lens.append(torch.tensor(len(trial_action), dtype=torch.float32))
+
+                # Publish the newly trained version every 100 steps
+                if len(actions) >= self._cfg.num_steps * self._cfg.epoch_num_trials + 1:
+                    num_updates += 1
+                    total_steps += self._cfg.num_steps * self._cfg.epoch_num_trials
+                    # Update model parameters
+                    policy_loss, value_loss = await self.train_step(
+                        observations=observations, rewards=rewards, actions=actions, dones=dones
+                    )
+
+                    # Reset the data storage
+                    observations = []
+                    actions = []
+                    rewards = []
+                    dones = []
+                    if iter_idx % 1 == 0:
+                        # Compute average rewards for last 100 episodes
```

Comment:
again a `100`

Reply:
Yep, I set 1 to debug. It now changes from 1 to 100

Reply:
make it configurable

Reply:
still need to make it configurable